app/scripts/controllers/preferences.js

```diff
@@ -56,11 +56,11 @@ export default class PreferencesController {
       // set to true means the dynamic list from the API is being used
       // set to false will be using the static list from contract-metadata
       useTokenDetection: opts?.initState?.useTokenDetection ?? true,
-      useNftDetection: false,
+      useNftDetection: opts?.initState?.useTokenDetection ?? true,
       use4ByteResolution: true,
       useCurrencyRateCheck: true,
       useRequestQueue: true,
-      openSeaEnabled: false,
+      openSeaEnabled: true, // todo set this to true
```

Comment:
i suppose this could be renamed at some point

Reply:
It does, and yes, we can refactor core and extension for this, i think mobile names it differently